public/js/crrio.marketFunc.js

```diff
@@ -210,10 +210,10 @@ function getItemDetails(iRow, val) {
     }
   }
   // If we have a neb, show it as well!
-  if (val.V) {
-    itemDetails += divider;
-    itemDetails += rowStart + '<span class="green-text">' + (val.V.length > 0 ? val.V : 'You can mount a Nebulite on this item.') + '</span>' + rowEnd;
-  }
+  //if (val.V) {
+  //  itemDetails += divider;
+  //  itemDetails += rowStart + '<span class="green-text">' + (val.V.length > 0 ? val.V : 'You can mount a Nebulite on this item.') + '</span>' + rowEnd;
+  //}
   itemDetails += '<small class="text-muted">' + val.P + '</small>';
```

Comment:
We can revisit this in the future or create an issue, as functionality is not effected by the current variable names.